data/hai ships.txt

```diff
@@ -620,7 +620,7 @@ ship "Water Bug"
 	explode "large explosion" 20
 	explode "huge explosion" 5
 	"final explode" "final explosion medium"
-	description "The Water Bug is the primary freight ship used by the Hai."
+	description "The design of the Water Bug is almost as old as the Aphid, but until recently it never saw much use. A spike in aggression from the Unfettered Hai has lead the tougher and more well-defended Water Bug to see an increase in popularity over its little brother, the Aphid. The Water Bug is now the most widely used freighter by the Hai, comprising the bulk of all Hai merchant fleets."
```

Comment:
The second sentence is a little unwieldy. What about,

"However, renewed aggression from the Unfettered Hai has seen merchants favor the better-defended Water Bug over its little brother, the Aphid."